build.sbt

```diff
@@ -17,14 +18,14 @@ inThisBuild(Seq(
     "-deprecation",
     "-Xfatal-warnings",
     "-encoding", "UTF-8",
-    "-release:11",
+    "-release:21",
```

Comment:
I'd also observe that this means Anghammarad can only be run locally by people running 21+, it gives the error "21 is not a valid choice for -java-output-version" if you're running 17 or 11, both common choices still.
I appreciate we do have a `.tool-versions` file here, but:
- I'm not sure how widely used that is, does everyone actually use asdf?
- the pinned version is currently 7 months out of date, but with no plan (or automation) this will get worse in the future, so we don't necessarily want to hang local development workflows off it

These are outstanding questions that we can sidestep by simple setting a more modest JDK target. Very happy to chat this over!